udata/forms/fields.py

```diff
@@ -285,6 +284,14 @@ def iter_choices(self):
             yield (value, label, selected)
 
 
+def slug(value):
+    return slugify(value.lower())
+
+
+def tags_list(value):
+    return list(set(slug(tag) for tag in value.split(',') if tag.strip()))
+
+
```

Comment:
Proposal:
- module `udata.tags`
- members:
  - `MIN_LENGTH`
  - `MAX_LENGTH`
  - `normalize`
  - `from_list`
(this only a proposal, names are up to you ;))